sandy/decay.py

```diff
@@ -171,6 +179,73 @@ def get_half_life(self, with_uncertainty=True):
         else:
             return df.HL
 
+    def custom_perturbation_half_life(self, pert, zam):
+        """
+        Apply a custom perturbation to the half life for a given ZAM.
+
+        Parameters
+        ----------
+        pert : `float`
+            Perturbation coefficient as a ratio value, e.g.,
+            1.05 for a perturbation of +5%
+        zam : `int`
+            ZAM number of the material to which perturbation is to be
+            applied.
+
+        Returns
+        -------
+        `sandy.decay.DecayData`
+            DacayData instance with given ZAM and half life perturbed.
+
+        Examples
+        --------
+        >>> endf6 = sandy.get_endf6_file("jeff_33", 'decay', [922350, 922380])
+        >>> rdd = sandy.DecayData.from_endf6(endf6)
+        >>> hl_pert = rdd.custom_perturbation_half_life(1.05, 922350)
+        >>> hl_pert.data
+        {922350: {'half_life': 2.332071e+16,
```

Comment:
this test is not really interesting.
I want to see that the half life has been perturbed by 5%